libs/community/langchain_community/tools/azure_ai_services/translate_tool.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
+
+import logging
+import os
+from typing import Any, Optional
+
+from azure.ai.translation.text import TextTranslationClient
+from azure.core.credentials import AzureKeyCredential
```

Comment:
Let's move these imports under the validate_environment and wrap them in a try/except so we can alert users if they need to install the library, for example:

https://github.com/Sheepsta300/langchain/blob/e5fb3635edc85b7233ebee1ebd58131a9513258d/libs/community/langchain_community/tools/azure_ai_services/content_safety.py#L55-L68